### PR TITLE
enable tmt tests through packit

### PR DESCRIPTION
**Pull request by sopos**

### CODE REVIEW

File: ci.fmf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+discover:
+    how: fmf
+    url: https://github.com/psss/tmt
+    ref: fedora
+    filter: 'tier: 1, 2 & framework:beakerlib'
+prepare:
+    how: install
+    package:
+    - tmt-all
+    - beakerlib
```
Comment (psss) [inline ci.fmf:16]: I would recommend to disable integration tests for `centos-7` as there is no `tmt` package there.

```yaml
adjust:
    enabled: false
    when: distro == centos-7
```

File: .packit.yaml [hunk 2]
```diff
@@ -29,3 +29,11 @@ jobs:
...
     - epel-8
     - epel-7
+
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-all
+    - epel-8
+    - epel-7
```
Comment (psss) [inline .packit.yaml:39]: Note that you don't need to specify both `tests` and `copr_build` jobs for pull requests unless you want to see separate test results for them. If only `tests` job is provided, copr builds are triggered automatically.

Comment (LecrisUT) [reply .packit.yaml:39]: If I'm not mistaking, recent updates require both test and copr_build to be defined

**Comment by sopos** [comment]
This was covered elsewhere, I guess.

**Pull request closed without merging**